### PR TITLE
Add warning about login with ORCID or institutional

### LINKED ISSUE #1108: Cannot Sign in to OSF on Jasp
* JASP version:0.14.1

* OS name and version:Mac Big Sur 11.1
* Analysis: Does not recognize my sign in to OSF
* Bug description:
* Expected behaviour:

### PR DESCRIPTION
**Pull request by JorisGoosen**
Fixes https://github.com/jasp-stats/jasp-issues/issues/1108

It now looks like:
image

Edit: the text is now "OSF account login only, ORCID or institutional login are not supported."

### CODE REVIEW

**Comment by JorisGoosen** [comment]
I saw @EJWagenmakers give a thumbsup for the text so Ive removed him as reviewer.

Now just wait for Tim's approval ^^

File: Desktop/components/JASP/Widgets/FileMenu/OSFLogin.qml
Comment (boutinb) [inline Desktop/components/JASP/Widgets/FileMenu/OSFLogin.qml]: I don't think a qsTr is necessary here. Translators will not know really what they have to do with this.

Comment (JorisGoosen) [reply Desktop/components/JASP/Widgets/FileMenu/OSFLogin.qml]: Copy paste all the way!

But you are right, I'll change it.

**Review by boutinb — APPROVED** [review]

**Review by TimKDJ — APPROVED** [review]

**Pull request merged**
